udata_gouvfr/theme/gouvfr/templates/dataset/resource/schema-button.html

```diff
@@ -1,5 +1,10 @@
 <button
-  class="btn btn-sm btn-primary"
-  @click.prevent="$showModal('schema', {resource_schema: {{resource.schema}}, documentation_url: '{{documentation_url}}', validation_url: '{{validation_url}}', authorize_validation: {{authorize_validation}} })">
+  class="btn btn-sm btn-primary hidden-md"
+  @click.prevent="$showModal('schema', {
+      resource_schema: {{resource.schema}},
+      documentation_url: '{{documentation_url}}',
+      validation_url: '{{validation_url}}',
+      authorize_validation: {{'true' if authorize_validation else 'false'}}
```

Comment:
authorize_validation is already a boolean, can't we keep with 
```suggestion
      authorize_validation: {{authorize_validation}}
```

Reply:
Nope because we need to switch from a Python boolean to a JS one (through a string ahah)